stage1/enterexec/enterexec.mk

```diff
@@ -1,3 +1,5 @@
-ASSCB_EXTRA_HEADERS := elf.h
+ASSCB_EXTRA_CFLAGS := -I$(MK_SRCDIR)/../diagnostic
+ASSCB_EXTRA_HEADERS := ../diagnostic/diagnostic-util.h
```

Comment:
We probably should include `../diagnostic/elf.h` here too, as it is included by diagnostic-util.h, right?